pepper-apis/src/main/java/org/broadinstitute/ddp/model/files/FileUploadStatus.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.ddp.model.files;
+
+public enum FileUploadStatus {
+    /**
+     * File upload has been authorized. This is the initial state.
+     */
+    AUTHORIZED,
+
+    /**
+     * File has been uploaded. File may not be safe to be operated on, and a scan is required/pending.
+     */
+    UPLOADED,
```

Comment:
These statuses will make more sense as we get around to implementing malware scanning. I didn't call this one `VERIFIED` since I worry folks will mistaken it to mean that it has been scanned and is safe, which it is not.